R/cml02a_gl.R

```diff
@@ -0,0 +1,50 @@
+# cml02a_gl_main ----
+
+#' @describeIn cml02a_gl Main TLG function
+#'
+#' @inheritParams gen_args
+#' @returns the main function returns an `rlistings` or a `list` object.
+#' @export
+#'
+cml02a_gl_main <- modify_default_args(std_listing,
```

Comment:
while reading `std_listing` in https://github.com/insightsengineering/chevron/blob/745303078cdc43d89886d9df7dd8980e20b371b0/R/utils.R#L66 I found that the `unique_rows` are set to `TRUE` by default. Please check if other listings should have this unique option

Reply:
so far, only `ael01_nollt` and `cml02a_gl` require `unique_rows = TRUE`. I have introduced this argument in the standard listing and set it to `FALSE` by default.